service/src/event/migration.rs

```diff
@@ -269,10 +263,13 @@ impl<'a> Migrator<'a> {
                 error!(%curr, "missing block");
                 break;
             }
+            // Add edge data to event
+            proof_edges.extend(edge);
```

Comment:
with this being after the above break the last iteration doesn't add the edge. Maybe that is the issue?